.buildkite/beta-builds.yml

```diff
@@ -32,15 +32,15 @@ steps:
   - group: "🚀 Beta Builds"
     steps:
 
-      - label: "🛠 WordPress Beta Build"
+      - label: ":wordpress: :android: Beta Build"
         key: wpbuild
         command: ".buildkite/commands/beta-build.sh wordpress"
         depends_on: wplint
         plugins: *common_plugins
         notify:
           - slack: "#build-and-ship"
 
-      - label: "🛠 Jetpack Beta Build"
+      - label: ":jetpack: :android: Beta Build"
```

Comment:
Since Gio [recently added Jetpack as an official emoji supported by Buildkite](https://github.com/buildkite/emojis/pull/392), I figured it was a great occasion to use it here to make the steps more readable:
<img width="498" alt="image" src="https://user-images.githubusercontent.com/216089/187194974-2e00b640-6131-4413-8b67-f7400b169226.png">

The main reason I did that change though was to make them more easily identifiable on the build notification in Slack (compared to previously where many other notifications used the same 🛠️  emoji and it was not as easy to know which app this was about just at a quick glance):
<img width="543" alt="image" src="https://user-images.githubusercontent.com/216089/187194609-3f1c70a7-0c67-4d61-ac42-a27e45e9b8bb.png">